web-app/js/portal/filter/DateFilter.js

```diff
@@ -123,17 +122,14 @@ Portal.filter.DateFilter = Ext.extend(Portal.filter.Filter, {
     _getDateString: function(newDate) {
 
         if (newDate) {
-            newDate.setHours(this.timezoneCorrection);
```

Comment:
No i cant. Someone like @mhidas or @ggalibert would be good at testing this. But they would need it merged to test it? Conundrum....